uber/templates/preregistration/preregbase.html

```diff
@@ -69,7 +71,7 @@
                 extra: 0
             }]
         };
```

Comment:
Hm, wait a minute, I'm not sure I like this. What if a staffer wants to pay for a second shirt? We put in logic last year explicitly to handle people who were entitled to two shirts, so I expect that is a relative common use-case.